openapi/extract_data_section.pl

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/perl
+use strict;
+use warnings;
+
+while ( my $file=shift(@ARGV)) {
+    my $yaml=$file;
+    $yaml =~ s/.+\///;
```

Comment:
This looks like a nice task for [File::Spec->splitpath()](https://perldoc.perl.org/File::Spec#splitpath).

Reply:
Rewritten